README.md

```diff
@@ -43,6 +43,7 @@
 
 - **[Create animals Database]**
 - **[Add insert data]**
+- **[Query and update animals table]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] The License link in the README file is broken, Please make sure to fix this. It will make the README file more professional.